### DISCUSSION TITLE
装了simhei和simsun之后，MacTex 编译出来的PDF字体与win 下的编译结果不一样。

### DISCUSSION THREAD

**Discussion by Hello-here** [whutug/whu-thesis#166] [Q&A] [answered]
装了**simhei**和**simsun**之后，**MacTex 编译出来的PDF字体**与**win 下的编译结果**不一样啊，有没有办法解决？

**Comment by xkwxdyy** [comment, marked as answer]
你「装了这个字体」 不代表 你「在 tex 中使用了此字体」

**Reply by Hello-here** [reply]
我是不是得自己看这个tex文件中字体调用的指令，然后修改？😂

**Reply by xkwxdyy** [reply]
其实吧，为什么非得用 win 的字体呢？难道 mac 电脑就不能写论文了吗，即使用 word 使用的仍然是 mac 的字体，缺失的话电脑会自行用其它字体代替，比如华文新魏。

**Reply by xkwxdyy** [reply]
> 我是不是得自己看这个tex文件中字体调用的指令，然后修改？

你有能力的话当然没问题。我在重写 whu-thesis，目前刚写完封面，字体打算最后再弄，反正是格式的东西，你倒可以先把重点放在内容上，八月底九月初应该是可以写完。到时候应该会配置一下字体。

**Reply by Hello-here** [reply]
哈哈哈，就怕毕业论文因为字体不对（即论文排版格式不对）被退回来，那就有大问题了。我是看到latex的同一个指令在win和mac上使用的字体不同。

**Reply by Hello-here** [reply]
好的，我先把写内容进去。目前这个模版在mac上编译的结果还是没有啥大问题，除了字体格式。谢谢啦。最后就算mac上格式不完全对，放到win上重新编译一下也可以。

**Reply by xkwxdyy** [reply]
> 最后就算mac上格式不完全对，放到win上重新编译一下也可以。

是的，这是个办法，反正周围一定有 win 电脑，到时候不放心的话，编译两份即可。我不觉得这是格式不对，这是 mac 和 win 调用的字体不同，但是基（我也不清楚术语是什么）应该是一样的，比如 mac 没有 Fangsong，但有 STFangsong

**Reply by Hello-here** [reply]
OK，谢谢大佬。哈哈哈

**Reply by Hello-here** [reply]
我尝试装完字体之后，修改了一下代码，好像搞定了。具体步骤如下：

1. 将Windows系统中的simhei.ttf、simsun.ttc、simkai.ttf、simfang.ttf拷贝到mac上
2. 将上面拷贝到mac上的字体进行安装：右键打开，然后弹出来的窗口点击安装，放心大胆地装
3. 将whu-thesis.cls文件中**% 使用 menlo 判断 mac，来自 ctex**这一部分的代码中**cjk-font  .initial:n = mac**改为**cjk-font  .initial:n = windows**
4. 提示一下，第三步中修改的指令后面有一个**英文逗号 “,” *

**Reply by xkwxdyy** [reply]
谢谢，之后会参考的。

**Reply by tanukihee** [reply]
> 我尝试装完字体之后，修改了一下代码，好像搞定了。具体步骤如下：
> 
> 1. 将Windows系统中的simhei.ttf、simsun.ttc、simkai.ttf、simfang.ttf拷贝到mac上
> 2. 将上面拷贝到mac上的字体进行安装：右键打开，然后弹出来的窗口点击安装，放心大胆地装
> 3. 将whu-thesis.cls文件中**% 使用 menlo 判断 mac，来自 ctex**这一部分的代码中**cjk-font  .initial:n = mac**改为**cjk-font  .initial:n = windows**
> 4. 提示一下，第三步中修改的指令后面有一个**英文逗号 “,” *

 都直接改源码了，为什么不愿意看看文档，直接用 `cjk-font = windows` 的接口呢……
